CombinatorialGames/Game/PGame.lean

```diff
@@ -162,41 +151,33 @@ lemma ext {x y : PGame} (hl : x.LeftMoves = y.LeftMoves) (hr : x.RightMoves = y.
 /-- Construct a pre-game from list of pre-games describing the available moves for Left and Right.
 -/
 def ofLists (L R : List PGame.{u}) : PGame.{u} :=
-  mk (ULift (Fin L.length)) (ULift (Fin R.length)) (fun i ↦ L[i.down.1]) fun j ↦ R[j.down.1]
+  mk (ULift (Fin L.length)) (ULift (Fin R.length)) (fun i ↦ L[i.down.1]) (fun j ↦ R[j.down.1])
 
+/-- Use `toOfListsLeftMoves` to cast between these two types. -/
+@[simp]
 theorem leftMoves_ofLists (L R : List PGame) : (ofLists L R).LeftMoves = ULift (Fin L.length) :=
   rfl
 
+/-- Use `toOfListsRightMoves` to cast between these two types. -/
+@[simp]
 theorem rightMoves_ofLists (L R : List PGame) : (ofLists L R).RightMoves = ULift (Fin R.length) :=
   rfl
 
-/-- Converts a number into a left move for `ofLists`.
-
-This is just an abbreviation for `Equiv.ulift.symm` -/
-abbrev toOfListsLeftMoves {L R : List PGame} : Fin L.length ≃ (ofLists L R).LeftMoves :=
```

Comment:
Did you mean to drop these abbrves when using referencing them in `left/rightMoves_ofLifsts`?

Reply:
These abbrevs aren't very useful, since the relevant type equality is definitional. That said... I think it's better to just close this PR. Presumably the plan is to port everything to `IGame`, and I think I've shown that `IGame` has no need for these type rewriting hacks.